src/app/ElectronBootstrap.js

```diff
@@ -248,12 +248,12 @@ module.exports = class ElectronBootstrap {
         }
 
         this._window = new electron.BrowserWindow({
```

Comment:
Keep the default width/height/title until there is a request by users to change them